app/Services/Course/StoreCourseService.php

```diff
@@ -17,10 +17,12 @@ class StoreCourseService
     public function __invoke(string $title, UploadedFile $image, int $tagId, int $instructorId): Course
     {
         // ファイルパスを作成
-        $file = $image;
-        $extension = $file->getClientOriginalExtension();
+        $extension = $image->getClientOriginalExtension();
         $filename = Str::uuid()->toString().'.'.$extension;
-        $filePath = Storage::putFileAs('public/course', $file, $filename);
+        $filePath = Storage::putFileAs('public/course', $image, $filename);
+        if (! $filePath) {
```

Comment:
PHPの場合、$filePathがfalsy（偽とみなされる値）」の場合にtrue になってしまいます。

- false
- null
- 空文字列 ""
- 数値の 0
- 空配列 []
- 文字列 "0"

そのため、意図している値を避ける場合は明示的に指定すると予期しないバグを防ぐことができます。

```suggestion
        if ($filePath === false) {
```